src/modules/ColorTemperature/Module.js

```diff
@@ -38,15 +38,15 @@ module.exports = function ColorTemperature(options, UI) {
       for (let i = 0; i < pixels.shape[0]; i++) {
         for (let j = 0; j < pixels.shape[1]; j++) {
 
-          r_data = pixels.get(i, j, 0);
+          var r_data = pixels.get(i, j, 0);
```

Comment:
the is a comma at the end so that the var can be shared with the other variable

Reply:
Hello, @aashna27 , could you please elaborate?